packages/terra-form/src/components/Input.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import './Input.scss';
+
+const propTypes = {
+  /**
+   * The defaultValue of the input field. Use this to create an uncontrolled input
+   */
+  defaultValue: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
+  /**
+   * Name of the input
+   */
+  name: PropTypes.string,
+  /**
+   * Functional to be called when the input is changed
+   */
+  onChange: PropTypes.func,
+  /**
+   * Whether the input is required or not
+   */
+  required: PropTypes.bool,
+  /**
+   * The value of the input field. Use this to create a controlled input
+   */
+  value: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
+};
+
+const defaultProps = {
+  defaultValue: undefined,
+  name: null,
+  onChange: () => {},
```

Comment:
I agree. The easiest way to do this is to remove onChange as a first class attribute from the component, as it comes default.

https://github.com/cerner/terra-core/pull/352/commits/41cbef501d514b9499cdb298b1576db3bdd7195a